### PR TITLE
Adding new cephx permissions chap (BSC#1157609)

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/admin_cephx.xml [hunk 1]
```diff
@@ -843,4 +843,73 @@ exported keyring for client.admin
    </variablelist>
   </sect2>
  </sect1>
+ <sect1 xml:id="permissions-cephx">
+   <title>Authorization Permissions (capabilities) for &cephx;</title>
+   <para>
+     &ceph; uses the term <literal>capabilities</literal> (caps) to describe
```
Comment (lproven) [inline xml/admin_cephx.xml:849]: I did not understand what "(caps)" meant. I was wondering if it was IN CAPITALS or something. I suggest spelling it out: "`capabilities`, or ''caps'' for short".

File: xml/admin_cephx.xml [hunk 2]
```diff
@@ -843,4 +843,73 @@ exported keyring for client.admin
...
+     authorizing an authenticated user to exercise the functionality of the
```
Comment (lproven) [inline xml/admin_cephx.xml:850]: "exercise the functionality of the"
... does that mean "use"?

File: xml/admin_cephx.xml [hunk 3]
```diff
@@ -843,4 +843,73 @@ exported keyring for client.admin
...
+     monitors, OSDs and metadata servers.
+   </para>
+   <para>
+     Capability syntax follows the form:
+   </para>
+<screen>
+{daemon-type} '{cap-spec}[, {cap-spec} ...]'
+</screen>
+   <para>
+     For example, monitor capabilities include <literal>r</literal>,
```
Comment (lproven) [inline xml/admin_cephx.xml:860]: This is quite a complex few paragraphs. Might I suggest a `variablelist` instead?

**Comment by asettle** [comment]
Sorry @lproven I ended up dismissing your review. Turns out I accidentally duplicated the section, so I merged the stuff I wrote with the old content. It's all here now, please take another look when you have time :)

**Pull request merged**